core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -235,7 +235,7 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
       _endedTasks.toSet
     }
 
-    private def waitForListeners(): Unit = sc.listenerBus.waitUntilEmpty()
+    private def waitForListeners(): Unit = sparkCtx.listenerBus.waitUntilEmpty()
```

Comment:
I'm thinking if we could have a new copy of `LocalSparkContext`, e.g., `LocalSC`, then, we don't need to change from `sc` to `sparkCtx`? And for other test suites, we only need to replace `LocalSparkContext` with `LocalSC`?

Reply:
OK